### DISCUSSION TITLE
How to make changes to a Custom domain page

### DISCUSSION THREAD

**Discussion by Voodazz** [community/community#47588] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have recently created a live static site using the Custom Domain feature.
My question is, how do I make changes to the sites files (index, css, etc) afterwards.
I pushed changes through the original repo via VSC , but that didn't do anything. 
Is there somewhere else I need to make these changes?

**Comment by InternazionaleMilano1908** [comment]
If you have created a live static site using the Custom Domain feature, then the site is most likely hosted on a static site hosting service, such as GitHub Pages or Netlify. To make changes to the site's files (index, css, etc.) after the initial deployment, you will need to update the files in the repository that is linked to the hosting service.

1. Make the desired changes to the files (index, css, etc.) in your local repository on your computer using an editor such as VSC.

2. Once you have made the changes, commit them to your local repository.

3. Push the changes to the remote repository that is linked to the hosting service. This will update the files in the repository with the changes you made locally.

4. Depending on the hosting service you are using, you may need to trigger a new build or deployment to see the changes on the live site. For example, on GitHub Pages, you may need to push the changes to a specific branch, such as the "gh-pages" branch, and then wait a few minutes for the changes to be deployed.
